docs/contributing/gatsby-docs-translation-guide.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Gatsby Docs Translation Guide
+---
+
+Since the [RFC on localization](https://github.com/gatsbyjs/rfcs/pull/42) for Gatsby, there is now an ongoing effort to translate the content on Gatsbyjs.org into multiple languages. For members of the community around the world building Gatsby sites and learning about web development, having docs and learning materials in your own language provides a lot of value. There's also a great opportunity for folks like you to contribute to the translations of the Gatsby docs.
```

Comment:
The RFC got merged so I think you can just link to the file: https://github.com/gatsbyjs/rfcs/blob/master/text/0010-gatsby-docs-localization.md

Reply:
Also, maybe we don't need to mention it? It seems like unnecessary context that could be confusing to people (eg 'what's an RFC?')

Reply:
Good idea! updated.